engine/hash_table.cpp

```diff
@@ -233,7 +233,7 @@ Status HashTable::SearchForRead(const KeyHashHint& hint, const StringView& key,
 void HashTable::Insert(
     const KeyHashHint& hint, HashEntry* entry_ptr, RecordType type, void* index,
     PointerType index_type,
-    HashEntryStatus entry_status /*= HashEntryStatus::Persist*/) {
+    KeyStatus entry_status /*= KeyStatus::Persist*/) {
```

Comment:
remove this strange comment.

Reply:
it is not strange comment, it reminds developer is default parameter. Some open source project often occurs.

Reply:
Default parameters are put in function declaration. Implementation does not care about that.